### DISCUSSION TITLE
BIRT How to print without preview

### DISCUSSION THREAD

**Discussion by MissingSlowly** [eclipse-birt/birt#1247] [Q&A]
when i want to print i find it should preview first ,but i just want print directedly

**Comment by wimjongman** [comment]
Please provide much more input. We can't do anything with this but guess your problem. Provide an extended description, screenshots, and movies.

**Comment by MissingSlowly** [comment]
给你深爱
***@***.***


When I want to use the print function of Birt, I need to jump to the preview window as shown below, and then click the print button. However, I do not want to preview now, and directly jump to the print interface when I print can you give me some advice







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "eclipse/birt"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月20日(星期一) 下午3:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [eclipse/birt] BIRT How to print without preview (Issue #1235)





 
Please provide much more input. We can't do anything with this but guess your problem. Provide an extended description, screenshots, and movies.

**Comment by hvbtup** [comment]
If you want to print, I suggest you use the PDF emitter, then print the PDF output.
PDF is made for printing.
